### PR TITLE
fix: update cypress design system buttons

### LINKED ISSUE #28852: Close button not working on Choose browser page v13.6.4
### Current behavior

Clicking close button closes the browser but relaunches it immediately after closing it.

### Desired behavior

Browser should stay closed after clicking Close button.

### Test code to reproduce

The issue is in the UI and reproduced by MikeMC399 and cy.david in discord [1].

[1] https://discord.com/channels/755913899261296641/1202942184542965823

### Cypress Version

v13.6.4

### Node version

v18.7.1

### Operating System

MacOS Sonoma 14.3 (23D56)

### Debug Logs

_No response_

### Other

The issue was not present at 13.6.2 according to discussion at discord (see above).

### PR DESCRIPTION
**Pull request by elevatebart**
closes #28852

Updates the vue button dependency to account for type property in html.
It fixes other issues with buttons being too large.

See [changelog](https://github.com/cypress-io/cypress-design/blob/main/components/Button/vue/CHANGELOG.md)

### What happens

When one clicks on the close Button, it has the wrong button type so, after closing the browser, the click re-submits the form, re-opening it. 

Why does it re-submit now while before this change everything was fine? Html buttons have `type="submit"` by default. and version 0.11.0 of the design system does not implement the type attribute properly.

### Why it happened

I made sure the bug was fixed in the design system and was implemented in full in my branch before moving the PR out of draft, but I expect that a bad merge rolled back the dependency to the button. As of now, the app and the launchpad still use 2 different version of the button. They are both outdated and with documented issues.

### How to fix

Update both dependency numbers to the latest version

### How to avoid it happening again

Add a **component test** to check that each button triggers the right event and only the right event.

### CODE REVIEW

**Review by MikeMcC399 — COMMENTED** [review]
@elevatebart 

I successfully tested on Ubuntu 22.04 and Windows 11 with Node.js 18.15.0 using `yarn` / `yarn start` with a default E2E project.

The Close button now closes the browser and does not reopen it. 👍🏻 

- The Changelog entry is missing. See https://github.com/cypress-io/cypress/actions/runs/7767770819/job/21184949310?pr=28860 and [Cypress App - Managing the Release Changelog](https://github.com/cypress-io/cypress/blob/develop/guides/writing-the-cypress-changelog.md). This should be added.

- CircleCI has not run any tests because the last commit was pushed whilst the PR was in Draft state. To trigger CircleCI, somebody with privileges should be able to this in https://app.circleci.com/pipelines/github/cypress-io/cypress?branch=28852-close-button-not-working-on-choose-browser-page-v1364 (I only have read-only access) or a dummy commit amend and force push to the branch, now it is no longer in Draft state would also trigger CircleCI.

**Review by MikeMcC399 — APPROVED** [review]
I successfully tested on Ubuntu 22.04 and Windows 11 with Node.js 18.15.0 using `yarn` / `yarn start` with a default E2E project.

The Close button now closes the browser and does not reopen it. 👍🏻

File: cli/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ _Released 2/13/2024 (PENDING)_
 
 - Improved accessibility of the Cypress App in some areas. Addressed in [#28774](https://github.com/cypress-io/cypress/pull/28774).
 
+**Bugfixes:**
+
+- LaunchPad: Stop reopening browsers after closing when clicking on close button. Fixes [#28852](https://github.com/cypress-io/cypress/issues/28852)
```
Comment (emilyrohrbough) [inline cli/CHANGELOG.md]: This needs to be moved above the misc section 
```suggestion
- Fixed an issue which caused the browser to relaunch after closing the browser from the Launchpad. Fixes [#28852](https://github.com/cypress-io/cypress/issues/28852).
```

Comment (elevatebart) [reply cli/CHANGELOG.md]: Isn't this a Bug fix? I do not understand.

Comment (emilyrohrbough) [reply cli/CHANGELOG.md]: The entire section needs to be moved above the misc section. See order of change type (bullet 3): https://github.com/cypress-io/cypress/blob/develop/guides/writing-the-cypress-changelog.md

**Comment by elevatebart** [comment]
@emilyrohrbough I updated the changelog AND moved it to Misc (though I fail to see the difference). 

I hope the PR is now OK.

**Comment by MikeMcC399** [comment]
@elevatebart 

> I updated the changelog AND moved it to Misc (though I fail to see the difference).

It seems there was a misunderstanding.

Bugfixes go in the Bugfixes section and the Bugfixes section is supposed to be before the Misc section. like in the release notes for `13.6.4`.

image

> The changelog does not include the **Bugfixes:** section. Given the pull request title provided, this section should be included in the changelog. If the changelog section is correct, please correct the pull request title to correctly reflect the change being made.

**Comment by elevatebart** [comment]
Thanks @MikeMcC399 for the pointer, I had skipped that part.

**Comment by elevatebart** [comment]
@emilyrohrbough sorry about my brain fart. 
I fixed the Changelog, moved the Bufixes section above the Misc stuff, and update the PR. I hope it can be merged soonish.

**Review by emilyrohrbough — APPROVED** [review]

**Review by mschile — APPROVED** [review]

**Pull request merged**